metric/system/process/process_linux_test.go

```diff
@@ -60,7 +61,11 @@ func TestFetchOtherProcessCgroup(t *testing.T) {
 	assert.NoError(t, err, "Init")
 
 	evts, _, err := testConfig.Get()
-	require.NoError(t, err)
+	if err != nil {
+		assert.True(t, errors.Is(err, NonFatalErr{}))
+	} else {
+		assert.NoError(t, err, "GetOne with one event")
+	}
```

Comment:
[Blocker]
This is always true, `if err == nil` calling `assert.NoError` is redundant.
same for the other tests.

Reply:
Argh, meant to check for `NotFatalError`